src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -106,29 +107,49 @@
     <Compile Include="$(PathToMsalSources)\Platforms\Features\DefaultOSBrowser\**\*.cs" />
     <Compile Include="$(PathToMsalSources)\Platforms\Features\DesktopOS\**\*.cs" />
     <Compile Include="$(PathToMsalSources)\Platforms\Features\OpenTelemetry\**\*.cs" />
+    <Compile Include="$(PathToMsalSources)\Platforms\Features\SLC\**\*.cs" />
+    <Compile Include="$(PathToMsalSources)\Platforms\Windows\**\*.cs" />
```

Comment:
I am not sure about this. Can you just make it part of "SLC" feature instead? We can't detect the operating system at build time only at runtime.